costmodel/costmodel.go

```diff
@@ -1378,6 +1378,12 @@ func (cm *CostModel) costDataRange(cli prometheusClient.Client, clientset kubern
 	}
 	clusterID := os.Getenv(clusterIDKey)
 
+	durHrs := end.Sub(start).Hours() + 1
+	durStr := fmt.Sprintf("%fh", durHrs)
```

Comment:
can we just use durHrs everywhere and divide ourself when we read logs, for simplicity?

Reply:
Yep, on it